tests/upgrade/test_crossgrade.sh

```diff
@@ -258,6 +260,22 @@ resetNamespaces() {
     done
 }
 
+# Returns 0 if the passed string has form "Code 200 : 6601 (94.6 %)" and the percentage is smaller than ${MIN_200_PCT_FOR_PASS}
```

Comment:
Wouldn't it be easier to extract only the integer part and compare with int 90?

Reply:
Yes it would, but I thought it was important to support floating as we may want to have high thresholds in the future e.g. 99.5%.